tmt/schemas/results.yaml

```
@@ -0,0 +1,58 @@
+---
+
+#
+# JSON Schema definition for tmt `results.yaml` file
+#
+# https://tmt.readthedocs.io/en/stable/spec/plans.html#execute
+#
+
+$id: /schemas/results
+$schema: https://json-schema.org/draft-07/schema
+
+type: array
+items:
+  type: object
+  additionalProperties: false
+
+  properties:
+    guest:
+      type: object
+      additionalProperties: false
+
+      properties:
+        name:
+          type: string
+
```

```suggestion
      pattern: "^/"

```

Should we add pattern specification for the `name` here?

---

Correct, that would be better.

For the sake of throughput, I'd fix that in a follow-up PR as this one is soooo close to being merged :)

---

Sorry, wrong line

---

Fixed in c2458f14.